src/test/java/com/rtb/projectmanagementtool/project/ProjectControllerTest.java

```diff
@@ -56,81 +56,80 @@ public void tearDown() {
-  public void testGetProjectWithUserAndCreator() {
-    // Only projects that USER1 created
-    // USER1: creates PROJECT1, creates PROJECT2, regularUser in PROJECT3
-    ArrayList<ProjectData> expectedUser1CreatorProjects =
-        new ArrayList<ProjectData>(Arrays.asList(PROJECT1, PROJECT2));
+  @Test
+  public void getProjectsWithUser() {
+    projectController.addProject(PROJECT1);
+    projectController.addProject(PROJECT2);
+    projectController.addProject(PROJECT3);
+    projectController.addProject(PROJECT4);
 
-    // Only projects that USER2 is in
-    // USER2: creates PROJECT3, admin in PROJECT2, regularUser in PROJECT4
-    ArrayList<ProjectData> expectedUser2Projects =
-        new ArrayList<ProjectData>(Arrays.asList(PROJECT2, PROJECT3, PROJECT4));
+    ArrayList<ProjectData> expectedUserProjects =
+        new ArrayList<ProjectData>(Arrays.asList(PROJECT3, PROJECT2, PROJECT4));
 
-    // Test retrieving projects USER1 created
-    ArrayList<ProjectData> actualUser1CreatorProjects = projectController.getCreatorProjects(USER1);
-    Assert.assertEquals(
-        expectedUser1CreatorProjects.toString(), actualUser1CreatorProjects.toString());
+    ArrayList<String> queryList = new ArrayList<String>();
+    queryList.add(ProjectData.createUserString(USER2, UserProjectRole.CREATOR));
+    queryList.add(ProjectData.createUserString(USER2, UserProjectRole.ADMIN));
+    queryList.add(ProjectData.createUserString(USER2, UserProjectRole.MEMBER));
 
-    // Test retrieving projects USER2 is in
-    ArrayList<ProjectData> actualUser2Projects = projectController.getUserProjects(USER2);
-    Assert.assertEquals(expectedUser2Projects.toString(), actualUser2Projects.toString());
+    ArrayList<ProjectData> actualUserProjects = projectController.getProjectsByQuery(queryList);
+    Assert.assertEquals(expectedUserProjects.toString(), actualUserProjects.toString());
   }
 
   // Test project removal from database
-  public void testRemoveProjects() {
-    // Check that number of projects = 4
-    Assert.assertEquals(4, projectController.getAllProjects().size());
+  @Test
+  public void removeProjects() {
+    projectController.addProject(PROJECT1);
+    projectController.addProject(PROJECT2);
+    projectController.addProject(PROJECT3);
+    projectController.addProject(PROJECT4);
 
-    projectController.removeProject(PROJECT1);
-    Assert.assertEquals(3, projectController.getAllProjects().size());
+    ArrayList<ProjectData> expectedProjects =
+        new ArrayList<ProjectData>(Arrays.asList(PROJECT1, PROJECT2, PROJECT3, PROJECT4));
+
+    Assert.assertEquals(expectedProjects.toString(), projectController.getAllProjects().toString());
 
+    expectedProjects.remove(PROJECT2);
     projectController.removeProject(PROJECT2);
-    Assert.assertEquals(2, projectController.getAllProjects().size());
+    Assert.assertEquals(expectedProjects.toString(), projectController.getAllProjects().toString());
```

Comment:
Junit should be able to assert the objects themselves, instead of converting to strings.  Maybe look into if you can assert these without writing to strings first.